observation_portal/accounts/views.py

```diff
@@ -102,3 +107,41 @@ def get_response_serializer(self, *args, **kwargs):
 
     def get_endpoint_name(self):
         return 'requestAccountRemoval'
+
+
+class BulkCreateUsersApiView(CreateAPIView):
+    """API endpoint to create users in bulk"""
+
+    permission_classes = [IsAdminUser]
```

Comment:
Just double checking, did you check in with Edward if his app has an admin account and could handle this (vs the PIs themselves sending these bulk create users requests?)

Reply:
No, I did not. I would assume only staff should be able to bulk create users to prevent spammers from creating a infinite number of accounts. Unless I'm misunderstanding, PI is a property of a Proposal and not a user role.

I guess we could create a custom permission class for users who have a membership in any Proposal as a PI and allow those users to hit the API too. But then the question again is do you trust all those users and how do you prevent them from creating a billion users each. Rate limiting should help, but we probably also have to have a hard quota. Or maybe I'm being too paranoid :)